RELEASING.md

```diff
@@ -8,16 +8,23 @@ Releasing checklist:
 
 (3) Checkout the `main` branch. This is the only releasable branch.
 
-(4) Create a tag for the release.
+(4) Create and push a release branch.
+
+```sh
+git checkout -b release-v0.12.0
+git push release-v0.12.0
+```
```

Comment:
Indeed. Maybe it would be better to rename it to `docs-v*`, and in cairo-contracts as well, as long as we use this branch only for the doc-site deployment. But that can be left for another issue.

Reply:
Yeah, I don't see any real problem with having using `docs-v*`.  Agreed on leaving it for another issue. It's not, by any means, pressing